tailwind.config.ts

```diff
@@ -0,0 +1,11 @@
+import type { Config } from 'tailwindcss';
+
+const config: Config = {
+  content: ['./index.html', './src/**/*.{js,ts,jsx,tsx}'],
```

Comment:
We don't really need these extensions
```suggestion
  content: ['./index.html', './src/**/*.{js,ts}'],
```